drivers/mtd_flashpage/mtd_flashpage.c

```diff
@@ -114,9 +114,13 @@ static int _write_page(mtd_dev_t *dev, const void *buf, uint32_t page, uint32_t
                 __attribute__ ((aligned (FLASHPAGE_WRITE_BLOCK_ALIGNMENT)));
 
         offset = addr % FLASHPAGE_WRITE_BLOCK_ALIGNMENT;
-        size = MIN(size, FLASHPAGE_WRITE_BLOCK_ALIGNMENT - offset);
+        size = MIN(size, FLASHPAGE_WRITE_BLOCK_SIZE);
```

Comment:
Why not
```suggestion
        size = MIN(size, FLASHPAGE_WRITE_BLOCK_SIZE - offset);
```

Reply:
I tried that it was wrong in some cases. I don´t remember in which cases. I will post an example later.

Reply:
I enabled the `DEBUG()` and tried an application with FLashDB. From the output it seems to be the same 🤔. I don´t know why I wrote it like this back then.